src/js/components/visualization/TableComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import changeCase from 'change-case'
+import {DataTable} from 'safe-framework'
+import Download from 'material-ui/svg-icons/file/file-download'
+import MenuItem from 'material-ui/MenuItem/MenuItem'
+import React, {Component, PropTypes} from 'react'
+
+export class TableComponent extends Component {
+  static propTypes = {
+    data: PropTypes.array.isRequired,
+    name: PropTypes.string.isRequired,
+    type: PropTypes.string.isRequired
+  }
+
+  static getToolbarItems (visualization) {
+    return [
+      <MenuItem
+        key='exportExcel'
+        leftIcon={<Download />}
+        primaryText='Export'
+        onTouchTap={function (event) {
```

Comment:
don't define callbacks inline. when you do, they get created on every render call. if we find this elsewhere in the code, it should be re-worked if possible